### PR TITLE
display & in titles nicely

### PR DESCRIPTION
**Pull request by grosser**
before:
screen shot 2017-09-06 at 5 05 44 pm

after:
screen shot 2017-09-06 at 5 04 41 pm

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -221,7 +221,9 @@ def additional_info(text)
 
   def page_title(content = nil, in_tab: false, &block)
     content ||= capture(&block)
-    content_for :page_title, strip_tags(content) + (@project ? " - #{@project.name}" : "")
+    title_content = strip_tags(content).gsub('&amp;', '&').html_safe
+    title_content << " - #{@project.name}" if @project
```
Comment (jonmoter) [inline app/helpers/application_helper.rb]: Is it possible for the project name to have a character like `&` in it?

Comment (grosser) [reply app/helpers/application_helper.rb]: stages can have that

Comment (grosser) [reply app/helpers/application_helper.rb]: ah I see ... yes we should do that after :D

Comment (grosser) [reply app/helpers/application_helper.rb]: fixed

**Pull request merged**